packages/calcite-components/src/components/progress/progress.scss

```diff
@@ -5,47 +5,52 @@
 *
 * These properties can be overridden using the component's tag as selector.
 *
-* @prop --calcite-progress-background-color: The background color of the component.
-* @prop --calcite-progress-color: The font color of the component.
-* @prop --calcite-progress-font-size: The font size of the component.
-*
-* @prop —-calcite-progress-label-inline-padding: Component label inline padding
-* @prop —-calcite-progress-label-block-end-padding: Component label block-end padding.
-* @prop —-calcite-progress-label-block-start-padding: Component label block-start padding.
+* @prop --calcite-progress-background-color: defines the background color of the component.
+* @prop --calcite-progress-bar-background-color: defines the background color of the progress bar.
```

Comment:
Maybe we can use `fill-color` here like in Meter PR instead of `bar-background-color`. Will also apply to Slider I'd imagine.

Reply:
yeah fill-color makes sense here.